python_modules/libraries/dagster-powerbi/dagster_powerbi/resource.py

```diff
@@ -137,9 +142,14 @@ def _fetch(
             "Authorization": f"Bearer {self._api_token}",
         }
         base_url = f"{BASE_API_URL}/groups/{self.workspace_id}" if group_scoped else BASE_API_URL
+        url = f"{base_url}/{endpoint}"
+        if params:
+            url_parameters = urlencode(params) if params else None
```

Comment:
nit: we don't need to check if params because it's the condition to enter the block.
```suggestion
            url_parameters = urlencode(params)
```